src/GHC/TypeLits/Extra/Symbol.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE UndecidableInstances #-}
+
+module GHC.TypeLits.Extra.Symbol (
+  StripPrefix,
+  ToLower,
+) where
+
+import GHC.TypeLits (ConsSymbol, Symbol, UnconsSymbol)
+
+-- | Strip `prefix` from `symbol`. Return `symbol` as-is if the prefix doesn't match.
+type family StripPrefix (prefix :: Symbol) (symbol :: Symbol) :: Symbol where
+  StripPrefix prefix symbol =
+    FromMaybe
+      symbol
```

Comment:
Can this be the empty string instead of the symbol itself? In most cases I want many routes to have no prefix, but if it uses `symbol` by default then this is not possible.

Reply:
Ah, I was thinking specifically is when writing `data R = R_Foo | Sub1 R1 | Sub2 R2` for `R1` and `R2` to be unprefixed, but arguably  `data R = R_Foo | Sub1 | Sub2` should fall back to `sub1.html` and `sub2.html` as you said